vocab.json

```diff
@@ -670,7 +670,55 @@
                 "english": ["modern"]},
                 {"id": 15,
                 "spanish": ["viejo", "antiguo", "antigua"],
-                "english": ["old"]}
+                "english": ["old"]},
+                {"id": 16,
+                "spanish":["la cocina"],
```

Comment:
I would usually like to see a space between the colon and the value.  It won't affect the functionality, it just keeps it a little tidier.